paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetActivity.kt

```diff
@@ -50,7 +50,11 @@ internal class CustomerSheetActivity : AppCompatActivity() {
 
         setContent {
             StripeTheme {
-                val bottomSheetState = rememberBottomSheetState()
+                val bottomSheetState = rememberBottomSheetState(
+                    confirmValueChange = {
+                        viewModel.bottomSheetConfirmStateChange()
```

Comment:
This method should only be called when the desired new state is `Hidden`.

Reply:
Ohh good catch